### ISSUE TITLE
Trouble with offset

### ISSUE THREAD

**Issue by wdonald1985** [stan-dev/rstanarm#86]
#### Summary:
A short summary (no more than a sentence or two).
I am fitting a generalized linear mixed model with a negbinom distribution and am having trouble with the offset

#### Description:
A full description of the problem (or question).
I have fit the model with lme4, brm, STATA, each of which has given me consistent coefficient estimates. However, when I fit the model in rstanarm I get different estimates. Additionally, I can even remove the offset completely and the estimates will be the same as when the offset is included.

arm <- stan_glmer.nb(Init_tot ~ numMax + (1|ID) + (1|Date), prior = normal(0,1), data = newData)
arm <- stan_glmer.nb(Init_tot ~ numMax + (1|ID) + (1|Date) + offset(offset), prior = normal(0,1), data = newData)
arm <- stan_glmer.nb(Init_tot ~ numMax + (1|ID) + (1|Date) + offset(log(offset)), prior = normal(0,1), data = newData)
arm <- stan_glmer.nb(Init_tot ~ numMax + (1|ID) + (1|Date) + offset(offset), link = "log", prior = normal(0,1), data = newData)

All of these give me the same estimates. However, removing the offset in other packages (eg. lme4) gives me about the same estimates as rstanarm when the offset is both included and excluded.

Lastly, if I remove the random effects from rstanarm, I get the same estimates as when they are included in the other packages.

###UPDATE
The rstanarm estimates with negbinom are similar to using lme4 with a gaussian distribution.
#### Reproducible Steps:
If applicable, the steps required to reproduce the issue. If you have a reproducible example, please include it.

#### RStanARM Version:
The version of the rstanarm package you are running (e.g., from `packageVersion("rstanarm")`)
newest
#### R Version:
The version of R you are running (e.g., from `getRversion()`)
3.2
#### Operating System:
Your operating system (e.g., OS X 10.11.3)
widows 7

**Comment by jgabry** [comment]
Thanks for reporting this. This is probably a bug, so we will look into
this as soon as possible and follow up once it's resolved or if we have any
questions.

**Comment by jgabry** [comment]
Ok I think this was an issue with specifying the offset inside the model formula rather than using the `offset` argument. They should be equivalent, but only the latter was working. I think this is fixed by my most recent commit (4aa2c3a), but I will do some further testing. If you want to try it out and see if it resolves the issue with your models you can install the latest version from github using 

```r
devtools::install_github("stan-dev/rstanarm", args = "--preclean")
```

**Comment by wdonald1985** [comment]
Hello:I installed the new version and it now works!thank you!Donny
 

    On Tuesday, March 22, 2016 1:18 PM, Jonah Gabry <notifications@github.com> wrote:
 

 Ok I think this was an issue with specifying the offset inside the model formula rather than using the offset argument. They should be equivalent, but only the latter was working. I think this is fixed by my most recent commit (4aa2c3a), but I will do some further testing. If you want to try it out and see if it resolves the issue with your models you can install the latest version from github using devtools::install_github("stan-dev/rstanarm", args = "--preclean")—

**Comment by jgabry** [comment]
Ok great thanks for letting us know. And thank you again for reporting the
bug.
